gradle.properties

```diff
@@ -22,3 +22,4 @@ org.gradle.configuration-cache=true
 android.useAndroidX=true
 android.enableAppCompileTimeRClass=true
 android.experimental.enableNewResourceShrinker.preciseShrinking=true
+android.experimental.additionalArtifactsInModel=true
```

Comment:
Why we need it?

Reply:
sry, some changes are not filtered when pushed, i revert it